conda-forge.yml

```
@@ -1,16 +1,17 @@
 build_platform:
+  linux_aarch64: linux_64
```

```suggestion
  linux_aarch64: linux_aarch64
```

It tries to use cross compilation, not emulation. Accept this change to use emulation. The reason is that travis has been backed up, so I think people tries to nudge feedstocks toward cross compilation (if they can) for speed, and sustainability.